### ISSUE TITLE
Proposal: Please start using Semantic Versioning

### ISSUE THREAD

**Issue by KateGo520** [leekchan/accounting#17]
I found that this project already [supports Go modules](https://github.com/leekchan/accounting/blob/v0.3/go.mod). But sadly, the [tags](https://github.com/leekchan/accounting/tags) doesn't follow [Semantic Versioning](https://semver.org), which means that all tags of this project will be ignored by Go modules and replaced by pseudo-versions, go get acts weirdly when tags are not in that form. It would be great to have the tagged release be named in the format vX.X.X format so that go mod can read it. 
```
$ go get github.com/leekchan/accounting@v0.3
go get github.com/leekchan/accounting@v0.3: no matching versions for query "v0.3"
$ go get github.com/leekchan/accounting
go: downloading github.com/leekchan/accounting v0.0.0-20191218023648-17a4ce5f94d4
go: github.com/leekchan/accounting upgrade => v0.0.0-20191218023648-17a4ce5f94d4
github.com/leekchan/accounting v0.0.0-20191218023648-17a4ce5f94d4
```
Else the mod file shows something like `github.com/leekchan/accounting v0.0.0-20191218023648-17a4ce5f94d4` which is not very readable and difficult to upgrade. It’s hard to verify which version is in use. This is not conducive to version control

So, I propose this project to follow Semantic Versioning in future versions. For example, `v1.0.1`, `v2.0.0`, `v3.1.0-alpha`, `v3.1.0-beta.2`etc.

**Comment by KateGo520** [comment]
@djackreuter  Could you help me review this issue? Thx :p

**Comment by leekchan** [comment]
Oh. Thank you. I've released new version v0.3.1. https://github.com/leekchan/accounting/releases/tag/v0.3.1 
I'll use semantic versioning 😄

**Comment by leekchan** [comment]
(Let me close this issue)
